OnBaseDocsApi/Controllers/OnBaseDocsController.cs

```diff
@@ -222,26 +222,35 @@ public IHttpActionResult ListDocs()
             var typeGroup = parms.Get("filter[typeGroup]", DefaultTypeGroup);
             var startDocId = parms.Get("filter[startDocId]", DefaultStartDocId);
             var pageSize = parms.Get("filter[pageSize]", DefaultPageSize);
+            // We bundle the bad requests in one response.
+            var badRequestErrors = new List<Error>();
+
+            if (string.IsNullOrWhiteSpace(indexKey) && string.IsNullOrWhiteSpace(typeGroup))
+                badRequestErrors.Add(BadRequestError($"At least one of filter[indexKey] or filter[typeGroup] is required."));
```

Comment:
I think this should be a normal string instead of the templated string.

Reply:
Good catch.